engines/director/detection_tables.h

```diff
@@ -3639,6 +3641,9 @@ static const DirectorGameDescription gameDescriptions[] = {
 	WINGAME1t_l("blinkybill1", "", "BLINKY95.EXE", "146512cdebe499a48872cc9e8710d738", 717025, Common::PT_BRA, 404),
 	MACGAME2_l(	"blinkybill1", "", "BB/16M", "17efee018a660458fae80de4364021ac", 481254,
 								   "A Caverna do Fantasma", "cbf7119d62a3528d490cd517b4451061", 5965, Common::IT_ITA, 404),
+	MACGAME2_l(	"blinkybill1", "", "BB/16M", 								"d:446bf208eda3b004bcf6afa840959775", 1189848,
+								   "xn--Das Geheimnis der Geisterhhle-55c", "d:ae26c0843073a479b8aacabca2a162bd", 6832, Common::DE_DEU, 404),
+	WINGAME1t_l("blinkybill1", "", "BLINKY95.EXE", "t:53dae92fa6a4f1ac8a95f5c1f5178d0f", 727923, Common::DE_DEU, 404),
```

Comment:
The checksum is prefixed with `t:` so you can just use `WINGAME1_l`

Reply:
Fixed and force-pushed.